### ISSUE TITLE
How to run the code with batchsize >1

### ISSUE THREAD

**Issue by ZhuYingJessica** [chenyuntc/simple-faster-rcnn-pytorch#181]
I'll be grateful if any one can give me some suggestions about how to run the code with batchsize greater than 1

**Comment by chrisway613** [comment]
in train.py, u can freely define the batch size in train().

**Comment by skyundead** [comment]
dataloader = data_.DataLoader(dataset, \
                                  batch_size=1, \
                                  shuffle=True, \
                                  # pin_memory=True,
                                  num_workers=opt.num_workers)

**Comment by chrisway613** [comment]
But I found if we set batch size > 1, it will raise an exception error..

**Comment by skyundead** [comment]
> 但是我发现如果我们将批处理大小设置为> 1，则会引发异常错误。

haha，me too。if batch_size = 2，it can‘t work。

**Comment by joqiaoy** [comment]
Have you solved this problem? I encountered this problem while training the model.

**Comment by ZhuYingJessica** [comment]
Sorry，i just changed another source code



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2022年03月24日 16:43 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [chenyuntc/simple-faster-rcnn-pytorch] How to run the code with batchsize >1 (#181) |

Have you solved this problem? I encountered this problem while training the model.

**Comment by Ayu000** [comment]
Is the cause of this problem the image size is not the same?

**Comment by ZhuYingJessica** [comment]
No，it seems that this code is not supportive for batchsize>1



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2022年05月02日 10:45 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [chenyuntc/simple-faster-rcnn-pytorch] How to run the code with batchsize >1 (#181) |

Is the cause of this problem the image size is not the same?
